packages/theming/demo/~patterns/stories/GetColorStory.tsx

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { StoryFn } from '@storybook/react';
+import styled, { useTheme } from 'styled-components';
+import { getColor } from '@zendeskgarden/react-theming';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Grid } from '@zendeskgarden/react-grid';
+import { Dots } from '@zendeskgarden/react-loaders';
+import { Code, Paragraph, Span } from '@zendeskgarden/react-typography';
+
+const StyledColor = styled.div`
+  display: inline-block;
+  border-radius: ${p => p.theme.borderRadii.md};
+  width: 100px;
+  height: 100px;
+`;
+
+export const GetColorStory: StoryFn = () => {
+  const theme = useTheme();
+  const [initialColor, setInitialColor] = useState(
+    getColor({ theme, variable: 'foreground.default' })
+  );
+  const [backgroundColor, setBackgroundColor] = useState(initialColor);
+  const [perf, setPerf] = useState({ milliseconds: 0, calls: 0 });
+  const BASELINE_NOCACHE = 2780;
+  const BASELINE_CACHE = 2000;
```

Comment:
While not critical for this update, we could discuss ways to improve the story without having to rely on a hardcoded BASELINE.